Makefile

```diff
@@ -29,6 +34,21 @@ tools: $(TOOLS)
 $(TOOLS): oci-%:
 	go build -ldflags "-X main.gitCommit=${COMMIT}" ./cmd/$@
 
+.PHONY: _man install uninstall
```

Comment:
I think *new* entries should go in the .PHONY at the end.  I don't have an opinion on collecting existing .PHONYs ;).